app/assets/javascripts/short-form/templates/b0a-prerequisites.html.slim

```diff
@@ -30,6 +30,8 @@
             small.error id="application_has_loan_pre_approvalError" ng-show="inputInvalid('application_has_loan_pre_approval')"
               | {{'LABEL.FIELD_REQUIRED' | translate}}
       lending-institution application="application" form="form"
+      .form-group.margin-left--2x.margin-right--2x
+        file-uploader button-label="{{'LABEL.UPLOAD_PREAPPROVAL_LETTER' | translate}}" file-label="{{'LABEL.PREAPPROVAL_LETTER' | translate}}" document="application.documents.preapprovalLetter" file-type="preapprovalLetter" required="true"
```

Comment:
should be `Loan pre-approval` here as the type I believe, so Salesforce will accept it. We should be sending "Document Type": "Loan pre-approval" through the API

Reply:
ok, will change it

Reply:
@akegan it's fixed